feature: tcc sample (#17)
CECTC/dbpack-samples: tcc/README.md

# dbpack-samples

Simply, you can build the environment by running the `make run` command. then, you can test by
- `curl -XPOST http://localhost:8080/transfer-commit` for transaction commit test, or
- `curl -XPOST http://localhost:8080/transfer-commit` for transaction rollback test.

> After execute `curl -XPOST http://localhost:8080/transfer-commit`, check log by `docker logs dbpack1` 

Reviewer: ```suggestion
After executing `curl -XPOST http://localhost:8080/transfer-commit`, check log by `docker logs dbpack1` 
```

> or `docker logs dbpack2`, then, you will see confirm interface be called automatically.

Reviewer: ```suggestion
or `docker logs dbpack2`, then, you will see confirm API been called automatically.
```

> After execute `curl -XPOST http://localhost:8080/transfer-cancel`, check log by `docker logs dbpack1`

Reviewer: ```suggestion
After executing `curl -XPOST http://localhost:8080/transfer-cancel`, check log by `docker logs dbpack1`
```

> or `docker logs dbpack2`, then, you will see cancel interface be called automatically.

Reviewer: ```suggestion
or `docker logs dbpack2`, then, you will see cancel API been called automatically.
```
